elements/pfe-readtime/test/pfe-readtime_test.js

```diff
@@ -0,0 +1,17 @@
+const elements = [...document.querySelectorAll("pfe-readtime")];
+
+suite("<pfe-readtime>", () => {
+
+    test("it should upgrade", () => {
+        assert.instanceOf(
+            document.querySelector("pfe-readtime"),
+            customElements.get("pfe-readtime"),
+            "pfe-readtime should be an instance of pfeReadtime"
+        );
+    });
+
+    // Write tests for each attribute
```

Comment:
Can you start by stubbing out some comments for what tests need to be written and we can pair program on how to write these?

Reply:
can do